scripts/_lib.sh

```diff
@@ -1,4 +1,30 @@
-npmCommand="yarn"
+npmCommand="npm"
+babelCliVersion="^6.18.0"
+
+function initialize {
+  path=${1-.}
+
+  # Install yarn to Meteor globals. Enable `meteor yarn` subcommand.
```

Comment:
Ok I decided to remove the automatic installation of `meteor yarn`.

`[ ! $(meteor npm list -gs --depth=0 | grep -o "yarn") ];`

That check causes an unnecessary delay that is repeated on each app and package installing. I hope they include it soon so that it won't be necessary for us to do it manually.